addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -103,17 +105,24 @@ var FormRenderer = BasicRenderer.extend({
         });
         return fieldNames;
     },
+    /*
+     * Updates translation alert fields for the current state and display updated fields
+     *
+     *  @param {Object} alertFields
+     */
+    updateAlertFields: function (alertFields) {
+        this.alertFields[this.state.res_id] = _.extend(this.alertFields[this.state.res_id] || {}, alertFields);
```

Comment:
no need for this.alertFields[this.state.res_id] =

The object this.alertFields[this.state.res_id] is already modified by _.extend

Reply:
Agree